beta-src/src/utils/map/drawCurrentMoveOrders.ts

```diff
@@ -1,21 +1,36 @@
 import TerritoryMap from "../../data/map/variants/classic/TerritoryMap";
 import ArrowType from "../../enums/ArrowType";
+import OrdersMeta from "../../state/interfaces/SavedOrders";
 import drawArrow from "./drawArrow";
 
-export default function drawCurrentMoveOrders(data): void {
+export default function drawCurrentMoveOrders(
+  data,
+  ordersMeta: OrdersMeta,
+): void {
   const { currentOrders, territories, units } = data;
-
-  if (currentOrders && units && territories) {
-    currentOrders
-      .filter((order) => order.type === "Move")
-      .forEach(({ id, toTerrID, unitID }) => {
-        const unitData = units[unitID];
-        const onTerrDetails = territories[unitData.terrID];
-        const toTerrDetails = territories[toTerrID];
-        const fromTerr = TerritoryMap[onTerrDetails.name].territory;
-        const toTerr = TerritoryMap[toTerrDetails.name].territory;
-        const arrowIdentifier = `${id}`;
-        drawArrow(arrowIdentifier, ArrowType.MOVE, toTerr, fromTerr);
+  const ordersMetaEntries = Object.entries(ordersMeta);
+  if (ordersMetaEntries.length && units && territories) {
+    ordersMetaEntries.forEach(([orderID, { update }]) => {
+      const originalOrder = currentOrders.find(({ id }) => {
+        return id === orderID;
       });
+
+      if (originalOrder) {
+        const { id, unitID } = originalOrder;
+
+        if (update) {
+          const { toTerrID, type } = update;
+          if (type === "Move" && toTerrID) {
+            const unitData = units[unitID];
+            const onTerrDetails = territories[unitData.terrID];
+            const toTerrDetails = territories[toTerrID];
+            const fromTerr = TerritoryMap[onTerrDetails.name].territory;
+            const toTerr = TerritoryMap[toTerrDetails.name].territory;
+            const arrowIdentifier = `${id}`;
```

Comment:
why does this have to be in a template string. Can't we just inline on L30 instead?

`drawArrow(id, ArrowType.MOVE, toTerr, fromTerr);`

Reply:
It doesn't, I think during the process of building this code I was trying out different things and left it this way.